app/main.py

```diff
@@ -2,0 +1,12 @@
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    command, source, destination = command.split()
+    if command != "mv":
+        return
+    path_, new_file_name = os.path.split(destination)
+    if path_ and not os.path.exists(path_):
+        os.makedirs(path_)
+    shutil.move(source, destination)
```

Comment:
Code Style: You are using the `os` module to manipulate paths. It's a good practice to use `os.path.join` to concatenate parts of the path. This will make your app cross-platform and will use either `/` or `\` depending on the OS.